PC02_Graffiti_COORS.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+#coding: utf-8
+
+"""
+PC02_Graffiti_Coors_McNulty.py
+ATLS 1300
+Author: Dr. Z
+Author: Cecily Coors
+September 8, 2020
+
+Graffiti of Jeff Bezos.
+"""
+
+from turtle import * #import turtle into workspace
+import math
+
+colormode(255)
+
+panel = Screen() #sets up the screen
+w=600
+h=600
+panel.setup(width=w,height=h) #changing size of my panel
+
+image="Bezos.gif"
+panel.bgcolor("lightgreen")
+panel.bgpic(image)
+
+#==== Add more code=====
+ht()
+t=Turtle()
+t=penup()
```

Comment:
```suggestion
t.penup()
```

Reply:
instead of setting your variable t equal to penup(), you want the turtle t to perform penup()
you can do this by using dot notation instead of using an equal sign, and should apply this change to the rest of your code